### PR TITLE
Feature/torques linear

### PR DESCRIPTION
**Pull request by gabrielenava**
This branch contains the latest version of the Matlab joint space controller and stack of task controller, where several bugs have been fixed and  useless functions and variables have been deleted. Also, there is the script for joint space linearization for the robot balancing on 1 foot.  Since most of the errors and bugs have been solved, I think it's time to merge this branch into the master.

### CODE REVIEW

**Comment by francesco-romano** [comment]
A description of the PR?

File: README.md [hunk 1]
```diff
@@ -13,13 +13,12 @@ To execute scripts that use `mex-wholebodymodel`, make sure that this directorie
...
 
-
-## Controller simulation
-In the [matlab-src/mex-wholebodymodel_balancing](matlab-src/mex-wholebodymodel_balancing/) and
-[matlab-src/TorqueBalancing_js](matlab-src/TorqueBalancing_js/) directories you can find two different
-simulations of a whole-body controller implemented using the mex-wholebodymodel interface.
-For more information please check the relative [README](matlab-src/mex-wholebodymodel_balancing/README),
-[README](matlab-src/TorqueBalancing_js/README).
+## Controllers simulations
+In [matlab-src/torqueBalancing_matlab](matlab-src/torqueBalancing_matlab/) the user can find two different
```
Comment (traversaro) [inline README.md]: Here we use "the user" why later we use "you". Let's be consider and always use you.

File: matlab-src/deprecated/README.txt [hunk 2]
```diff
@@ -1,8 +0,0 @@
-More detailed documentation and comments throughout the code will be provided soon.
-
-To run the simulator, run the 'run_forwardDyn.m' file. User can change various parameters of the simulation by modifying 'set_parameters.m' file which can also be accessed from the GUI. GUI gives the options to set simulation duraion and step times which by default are set to 10 and 0.01 seconds relatively. Also the checkbox 'Plots' allows the plotting of the contact forces at the feet, joint torques and center of mass trajectory errors.
```
Comment (traversaro) [inline matlab-src/deprecated/README.txt]: In which file this documentation of the forwardDynamics was ported ?

File: matlab-src/jointSpaceLinearization/linearization1Foot_base.m [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+%  the robot balancing on one foot.
+%
+clear all
+close all
+clc
+
+%% Path and model initialization
+addpath('./../../mex-wholebodymodel/matlab/utilities');
+addpath('./../../mex-wholebodymodel/matlab/wrappers');
+addpath('./../../../../build/');
```
Comment (traversaro) [inline matlab-src/jointSpaceLinearization/linearization1Foot_base.m]: This should be avoided in the code (see https://github.com/robotology/mex-wholebodymodel/issues/17). You should use a proper `pathdef.m` file or `MATLABPATH` enviromental variable. 

Feel free to ask me more details.

File: matlab-src/jointSpaceLinearization/linearization1Foot_centroidal.m [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+%  in centroidal coordinates.
+%
+clear all
+close all
+clc
+
+%% Path and model initialization
+addpath('./../../mex-wholebodymodel/matlab/utilities');
+addpath('./../../mex-wholebodymodel/matlab/wrappers');
+addpath('./../../../../build/');
```
Comment (traversaro) [inline matlab-src/jointSpaceLinearization/linearization1Foot_centroidal.m]: See https://github.com/robotology/mex-wholebodymodel/pull/65/files#r55707656 .

File: mex-wholebodymodel/matlab/utilities/jointLimits.m [hunk 5]
```diff
@@ -0,0 +1,5 @@
+%% Joint limits definition for icubGazeboSim (rad)
```
Comment (traversaro) [inline mex-wholebodymodel/matlab/utilities/jointLimits.m]: You should use the `wbm_jointLimits` function in place of this hardcoded limits.

File: matlab-src/deprecated/README.txt (hunk 2 above)
Comment (gabrielenava) [inline matlab-src/deprecated/README.txt]: There are two readmes for the controllers in master, but I already tought to rewrite them after the merge

File: mex-wholebodymodel/matlab/utilities/whatname.m [hunk 6]
```diff
@@ -0,0 +1,163 @@
+function name = whatname(name_group,k)
+%% whatname
+%  It assigns titles to graphics according to robot's joints distribution.
+%  The inputs are the robot link name (torso, r_arm, l_arm,...) and the interested
```
Comment (traversaro) [inline mex-wholebodymodel/matlab/utilities/whatname.m]: There is already a convention of the iCub part names, that are the one used in the controlboards, see http://wiki.icub.org/wiki/ICub_joints . If possible, use `right_arm` and `left_arm` in place of `l_arm` and `r_arm`.

File: matlab-src/deprecated/README.txt (hunk 2 above)
Comment (traversaro) [inline matlab-src/deprecated/README.txt]: Ok!

**Pull request merged**